src/Utils.ts

```diff
@@ -122,6 +125,23 @@ class Utils {
         transactions.push(transaction);
         localStorage.setItem(address, JSON.stringify(transactions));
     }
+
+    static async setBalance(
```

Comment:
I think this method should be called `getSmartWalletBalance` or `getTokenBalances`